charts/delphai-keda/templates/deployment.yml

```diff
@@ -61,7 +61,7 @@ spec:
           {{- end }}
         {{ end }}
       containers:
-        - image: {{ .Values.image }}
+        - image: {{ required "`image` value is required" .Values.image }}
```

Comment:
👍

Reply:
To avoid a case when [projects](https://github.com/delphai/helm-charts/pull/39/files#diff-f49b92630374e41fde356d82bfdcc2a8e8cdd069530b90819e99484cfc194600L1) service was deployed by default